### ISSUE TITLE
WARNING: eval from module Base to 'my_package name'

### ISSUE THREAD

**Issue by goedman** [JuliaDocs/Documenter.jl#806]
I asked about this a few weeks ago on Discourse [Usage] and thought it was resolved in Documenter 0.19.5 but it isn't. Not sure if I'm doing something wrong or if below warning is just a consequence from moving to Julia 0.7 and 1.0:
```
WARNING: eval from module Base to CmdStan:    
Expr(:->, :x, Expr(:block, #= Symbol("/Users/rob/.julia/packages/Documenter/bK0Xg/src/DocChecks.jl"):651 =#, :y = :have_color, #= Symbol("/Users/rob/.julia/packages/Documenter/bK0Xg/src/DocChecks.jl"):651 =#, Expr(:global, :have_color = :x), #= Symbol("/Users/rob/.julia/packages/Documenter/bK0Xg/src/DocChecks.jl"):651 =#, :y))
  ** incremental compilation may be broken for this module **
```

Any insights or tips appreciated.

**Comment by fredrikekre** [comment]
Do you have a branch where this happens?

**Comment by goedman** [comment]
It happens the first time I run a documented package in a session (and often after a restart if the package is recompiled?), i.e.

```
(0.7) pkg> add CmdStan
(0.7) pkg> test CmdStan
```

The message is displayed before the testing even begins.

I developed these docs over a few years, so it might be caused by something outdated I do. But it seems to me it must come from an @eval or so.

**Comment by goedman** [comment]
Indeed, I have commented out that line in __init__ and haven't seen the message again. I'll keep trying for the next few days, but it seems your suggestion solves it. Thanks a lot!
